apps/web/src/server/app/validation/validation.controller.js

```diff
@@ -273,6 +309,11 @@ export function getCheckAndConfirm(request, response) {
 	const appealData = request.session.appealData;
 	const appealWork = request.session.appealWork;
 
+	let invalidReasons;
+	if (appealWork.invalidAppealDetails && appealWork.invalidAppealDetails.invalidReasons) {
+		invalidReasons = arrayifyIfString(appealWork.invalidAppealDetails.invalidReasons);
```

Comment:
This will need to be replaced with `invalidReasons = flatten([appealWork.invalidAppealDetails.invalidReasons])`